lib/settings-panel.coffee

```diff
@@ -189,6 +189,9 @@ appendSetting = (namespace, name, value) ->
     return if name is 'disabledPackages' # Handled in the Packages panel
     return if name is 'customFileTypes'
 
+  if namespace is 'editor'
+    return if name in ['commentStart', 'commentEnd', 'increaseIndentPattern', 'decreaseIndentPattern', 'foldEndPattern'] # There's no global default for these, they are defined by language packages
```

Comment:
Super minor, but I think putting the comment above this line instead of at the end might make a bit more readable.